### ISSUE TITLE
mms

### ISSUE THREAD

**Issue by tony5** [andyholmes/gnome-shell-extension-mconnect#17]
is it possible to receive media in messages? or just text?
take care!

**Comment by andyholmes** [comment]
Unfortunately the state of telephony support in KDE Connect is pretty limited right now.

The only outgoing telephony based function is "sendSms" which takes a phone number and a text field.

Incoming telephony is actually handled through the notifications plugin, and each notification has the properties _"dismissable"_, _"appName"_, _"iconPath"_ (not used/always empty?), _"internalId"_ (eg com.google.android.apps.messaging) and _"ticker"_ (a text-only field). If you're interested in the nitty gritty of what KDE Connect can do, install [D-Feet](https://wiki.gnome.org/Apps/DFeet) and search for "connect" in the "Session Bus" tab and you can investigate everything available for KDE Connect or MConnect.

TL;DR Nope, just text :(
